pom.xml

```diff
@@ -376,8 +376,8 @@
         <test.elasticsearch.version>${version.org.elasticsearch.latest}</test.elasticsearch.version>
 
         <test.elasticsearch.run.elastic.skip>true</test.elasticsearch.run.elastic.skip>
-        <test.elasticsearch.run.elastic.image.name>elastic/elasticsearch</test.elasticsearch.run.elastic.image.name>
-        <test.elasticsearch.run.elastic.image.tag>${test.elasticsearch.version}</test.elasticsearch.run.elastic.image.tag>
+        <test.elasticsearch.run.elastic.image.name>docker.elastic.co/elasticsearch/elasticsearch</test.elasticsearch.run.elastic.image.name>
+        <test.elasticsearch.run.elastic.image.tag>master-SNAPSHOT</test.elasticsearch.run.elastic.image.tag>
```

Comment:
yes 😃 🙈  (`[HSEARCH-4920 TEST DON'T MERGE]`)

Reply:
ok so that seems to work:
```
[INFO] DOCKER> Pulling from elasticsearch/elasticsearch
[INFO] DOCKER> Digest: sha256:f201f483c2d97aea6f588990118ac32d4cbfbc0423b3a0a3885efa98d4e3f64c
[INFO] DOCKER> Status: Downloaded newer image for docker.elastic.co/elasticsearch/elasticsearch:master-SNAPSHOT
[INFO] DOCKER> Pulled docker.elastic.co/elasticsearch/elasticsearch:master-SNAPSHOT in 23 seconds 
[INFO] DOCKER> [docker.elastic.co/elasticsearch/elasticsearch:master-SNAPSHOT] "elasticsearch": Start container 9e9ffae99d94
[INFO] DOCKER> [docker.elastic.co/elasticsearch/elasticsearch:master-SNAPSHOT] "elasticsearch": Waiting on url http://localhost:9200 with method GET for status 200.
```
I'm going to revert that test commit now 😃